src/service/video-manager-impl.js

```diff
@@ -325,6 +364,44 @@ class VideoEntry {
   }
 
   /**
+   * Called when the orientation of the device changes
+   * @private
+   */
+  orientationChanged_() {
+
+    const vidElement = this.video.element.querySelector('iframe, video');
+
+    // Determine which implementation of the orientation angle to use
+    const angle = this.ampdoc_.win.orientation !== undefined ?
+                  this.ampdoc_.win.orientation : screen.orientation.angle;
+
+    if (this.isVisible_
+    && this.getPlayingState() == PlayingStates.PLAYING_MANUAL
+    && (Math.abs(angle) == 270 || Math.abs(angle) == 90)) {
+
+      // Determine which implementation of requestFullScreen to use
+      const requestFullScreen = vidElement.webkitEnterFullscreen
```

Comment:
let's refactor to two new methods on `entry`: `enterFullscreen_()` and `exitFullscreen_()`